app/models/workflow_execution.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# entity class for Sample
+class WorkflowExecution < ApplicationRecord
+  has_logidze
+  acts_as_paranoid
+
+  belongs_to :submitter, class_name: 'User'
+
+  serialize :metadata, coder: WorkflowMetadata
+  validates_with WorkflowMetadataValidator
```

Comment:
Instead of serializing this as `WorkflowMeta` and your custom validator can you change it use a json schema. See this gem https://github.com/mirego/activerecord_json_validator

Reply:
fixed in https://github.com/phac-nml/irida-next/pull/308/commits/b93ceec0d2aa3386add4aafbf34c30d56610ee7f